chemicaltoolbox/rdkit/sdf_to_tab.xml

```diff
@@ -0,0 +1,104 @@
+<tool id="sdf_to_tab" name="Extract values from an SD-file" version="@VERSION@">
+    <macros>
+        <token name="@VERSION@">2019.03.1</token>
+    </macros>
+    <description>into a tabular file using RDKit</description>
+    <requirements>
+        <requirement type="package" version="@VERSION@">rdkit</requirement>        
+    </requirements>
+    <command detect_errors="exit_code">
+<![CDATA[
+        python '$__tool_directory__/sdf_to_tab.py'
+            -i '${infile}'
+            -o '${outfile}'
+            -p '$props'
+            $header
+]]>
+    </command>
+    <inputs>
+        <param name="infile" format="sdf" type="data" label="Input SD-file"/>
+        <param name="header" type="boolean" label="Include the property name as header"
+            truevalue="--header" falsevalue="" checked="false" />
+        <param name="props" type="text" label="Properties to extract from the SD-file" help="Separated with a comma, e.g. RMSD_UB,RMSD_LB,SCORE. To extract values for all properties in the file, leave the field blank."/>
```

Comment:
So this should be optional=True rihgt?

Reply:
Originally I wanted to use `multiple="true"` but this doesn't work for text params. I don't know if there is some more officially favoured solution?